src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+
+        <label class="field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+
+        <label class="field">
+          I accept the term of the agreement
+          <input type="checkbox" name="acceptedTerm">
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
```

Comment:
point 3 in [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)